src/test/ui/traits/trait-alias/trait-alias-no-duplicates.stderr

```diff
@@ -0,0 +1,358 @@
+error[E0225]: only auto traits can be used as additional traits in a trait object
+  --> $DIR/trait-alias-no-duplicates.rs:16:22
+   |
+LL | trait _0 = Obj;
+   |            ---
+   |            |
+   |            additional non-auto trait
+   |            first non-auto trait
+...
+LL | type _T00 = dyn _0 + _0;
```

Comment:
Can you do the "referenced here" diagnostic for the "first non-auto trait", too?

Reply:
Yeah, I discussed this with @Centril before and it leads to some pretty confusing error messages when the expansion paths coincide for the first few levels. I'll try differentiating the error messages by suffixing them with ` (first)` and ` (additional)` or something. That was the alternative we discussed.